.storybook/preview-body.html

```diff
@@ -51,6 +51,46 @@
   .innerZoomElementWrapper > div > div > div {
     min-height: 300px !important;
   }
+
+  .sb-decorator-example-scroller {
```

Comment:
@Dietr Is it possible to place this styling inside a SCSS file that is only used within Storybook? I'm not sure this is the correct place for it.

Reply:
So far I have not found a better way of handling css overrides in storybook.

Not sure what this does exactly but maybe it's also possible to do in the ember test application? There it's easier to manage styles.